### PR TITLE
SIMSBIOHUB-588 Capture

### PR DESCRIPTION
**Pull request by MacQSL**
Related to: [SIMS 588 Capture](https://github.com/bcgov/biohubbc/pull/1305)

### CODE REVIEW

File: prisma/migrations/20240604213907_capture_method_changes/migration.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `capture_timestamp` on the `capture` table. All the data in the column will be lost.
+  - You are about to drop the column `release_timestamp` on the `capture` table. All the data in the column will be lost.
+  - Added the required column `capture_date` to the `capture` table without a default value. This is not possible if the table is not empty.
```
Comment (NickPhura) [inline prisma/migrations/20240604213907_capture_method_changes/migration.sql:6]: Just checking: presumably not an issue for current critterbase dev/test/prod?

File: prisma/seed_scripts/seed_critters.ts [hunk 2]
```diff
@@ -1,40 +1,37 @@
-import { prisma } from "../../src/utils/constants";
-import { CRITTER_ONE, ITIS_TSN } from "./seed_constants";
+import { prisma } from '../../src/utils/constants';
+import { ITIS_TSN, STATIC_UUIDS } from './seed_constants';
 
 export const seedCritters = async () => {
-  console.log("Seeding (1) critter + attributes...");
+  console.log('Seeding (1) critter + attributes...');
```
Comment (NickPhura) [inline prisma/seed_scripts/seed_critters.ts:5]: 👀

File: prisma/migrations/20240604213907_capture_method_changes/migration.sql (hunk 1 above)
Comment (MacQSL) [reply prisma/migrations/20240604213907_capture_method_changes/migration.sql:6]: Should be ok 👍🏼

**Review by NickPhura — APPROVED** [review]
🌵

**Pull request merged**